android_mic_streaming/README.md

```diff
@@ -0,0 +1,53 @@
+# Android Microphone Streaming
+
+Android demo application that streams audio from the microphone to deepspeech and transcribes it.
+
+## Prerequisites
+
+#### Download model
+
+Download the pre-trained English model and extract it:
+```
+curl -LO https://github.com/mozilla/DeepSpeech/releases/download/v0.6.1/deepspeech-0.6.1-models.tar.gz
+tar xvf deepspeech-0.6.1-models.tar.gz
+```
+
+Move the model files `output_graph.tflite`, `lm.binary`, `trie` to the demo application's data directory on your android device.  
+Mind that the data directory will only be present after installing and launching the app once.
+
+```
+cd deepspeech-0.6.1-models
+adb push output_graph.tflite lm.binary trie /storage/emulated/0/Android/data/org.deepspeechdemo/files/
+```
+
+You can also copy the files from your file browser to the device.
+
+#### Android device with USB Debugging
+
+Connect an android device and make sure to enable USB-Debugging in the developer settings of the device. If haven't already, you can activate your developer settings by following [this guide from android](https://developer.android.com/studio/debug/dev-options#enable).
```

Comment:
Technically you will also have to authorize the computer, but a first `adb shell` should prompt that.